docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+
+
+## [Multi-start optimisation](@id petab_multistart_optimisation)
+To avoid the optimisation process returning a local minimum, it is often advised to run it multiple times, using different initial guesses. PEtab.jl supports this through the `calibrate_model_multistart` function. This is identical to the `calibrate_model` function, but takes two additional arguments:
+
+1. `n_multistart`: The number of runs to perform.
+2. `dir_save`: A location to which the output is saved. If `dir_save=nothing`, no output is saved.
+
+And one additional optional argument:
+
+3. `sampling_method`: Selects the sampling method with which to select the initial guesses (`QuasiMonteCarlo.LatinHypercubeSample()` used by default). 
+ 
+Because `calibrate_model_multistart` handles initial guess sampling, unlike for `calibrate_model`, no initial guess has to be provided. 
+
+Here, we fit parameters through 10 independent optimisation runs, using QuasiMonteCarlo's `SobolSample` method, and save the result to the OptimizationRuns folder:
+```
+using Optim
+using QuasiMonteCarlo
+res_ms = calibrate_model_multistart(petab_problem, IPNewton(), 10, "OptimizationRuns"; sampling_method=QuasiMonteCarlo.SobolSample())
+```
+The best result across all runs can still be retrieved using `get_ps(res_ms, petab_problem)`, with the results of the individual runs being stored in the `res_ms.runs` field. 
+
+To load the result in a later session, we can call:
+```
+res_ms = PEtabMultistartOptimisationResult("OptimizationRuns")
+```
+If the OptimizationRuns folder contains the output from several runs, we can designate which to load using the `which_run` argument. Here we loads the second run to be saved in that folder:
+```
+res_ms = PEtabMultistartOptimisationResult("OptimizationRuns"; which_run="2")
+```
+
+
+## [Events](@id petab_events)
+So far, we have assumed that all experiments, after initiation, run without interference. Experiments where conditions change, or where species are added/removed during the time course, can be represented through events (related to [callbacks](@ref advanced_simulations_callbacks)). In PEtab, an event is represented through the `PEtabEvent` structure. It takes three arguments:
+1. The condition for triggering the event. This can either indicate a point in time, or a boolean condition.
+2. A rule for updating the event's target
+3. The event's target (either a species or parameter).
+
+Here we create an event which adds `0.5` units of `S` to the system at time `5.0`:
+```petab1
+@unpack S = rn
+event1 = PEtabEvent(5.0, S + 0.5, S)
+```
+Here, the first argument is evaluated to a scalar value, in which case it is interpreted as a time point at which the event happens. If we instead want the event to happen whenever the concentration of `S` falls below `0.5` we set the first argument to a boolean condition indicating this:
```

Comment:
Maybe a bit advanced, but might be worth out to point out that for Boolean events like these they are only triggered when transition from `false` to `true`, so `S < 0.5` is only triggered as written when falling below 0.5 (from above), whether `S == 0.5` would be activated when coming from both direction

Reply:
Just checking, bu this means that if the event continues being true (e.g. when S <0.5 we do something entirely different) the event does not trigger. It only triggers when the transition occur?

Reply:
Exactly, it only triggers when we make transition from `false` to `true` (so when the statement already is true nothing happens trigger wise)

Reply:
added additional explanation.